app/services/charges/apply_taxes_service.rb

```diff
@@ -13,6 +13,10 @@ def call
       return result.not_found_failure!(resource: 'charge') unless charge
       return result.not_found_failure!(resource: 'tax') if (tax_codes - taxes.pluck(:code)).present?
 
+      charge.applied_taxes.where(
+        tax_id: charge.taxes.where.not(code: tax_codes).pluck(:id),
```

Comment:
Should we keep the `PaperTrailTraceable` on `Charge::AppliedTax`?

If we remove everything at each update, I have the feeling that it will be useless. What do you think?